### DISCUSSION TITLE
Auto connect don't work like I set up

### DISCUSSION THREAD

**Discussion by AnonymHuman** [d4rken-org/capod#275] [Q&A]
Auto connect don't work right for me. I selected "if seen" but it tries to connect when the case is closed with the AirPods inside. And if I want use my AirPods, I open the case but it don't want to connect.

(Android 14, OneUI 6.0; CaPod Pro)

**Comment by d4rken** [comment]
Do you have a debug log of this?

CAPod tries to start the connection once the AirPods are "seen", if the AirPods are not seated correctly in the case, and they are still sending data (not in standby) then this current counts as "being seen".
